app/services/record_declarations/actions/make_declarations_payable.rb

```diff
@@ -4,13 +4,13 @@ module RecordDeclarations
   module Actions
     class MakeDeclarationsPayable
       class << self
-        def call(start_date: "2021-09-01 00:00:00", end_date: "2021-11-01 00:00:00")
-          new.call(start_date: start_date, end_date: end_date)
+        def call(declaration_type:, start_date: "2021-09-01 00:00:00", end_date: "2021-11-01 00:00:00")
```

Comment:
I think this is a bit confusing as `ParticipantDeclaration` has a `declaration_type` field. `declaration_klass` ?

Reply:
@jamesrose it's stored as the `type` in the the db. also could just be `declaration_class`